clang/cmd.go

```diff
@@ -6,9 +6,8 @@ import (
 	"os"
 	"strings"
 
+	"github.com/go-clang/gen"
```

Comment:
I am a bit cumbersome when it comes to imports. I prefer to group then in three groups:
- STD imports
- third party imports
- project imports